src/components/utils/popover/popover-desktop.ts

```diff
@@ -160,15 +163,25 @@ export class PopoverDesktop extends PopoverAbstract {
     super.destroy();
   }
 
+  /**
+   * Handles displaying nested items for the item.
+   *
+   * @param item – item to show nested popover for
+   */
+  protected override showNestedItems(item: PopoverItemDefault): void {
+    if (this.nestedPopover !== null && this.nestedPopover !== undefined) {
+      return;
+    }
+    this.showNestedPopoverForItem(item);
+  }
+
   /**
    * Handles input inside search field
```

Comment:
Please, update jsdoc for this method. It's not clear what it is doing for now. Do we still need it?